### PR TITLE
Don't drop colors on active links

### PR DESCRIPTION
**Pull request by NotMyFault**
The change proposed keeps up colors on links and possible ionicons placed on the link, like it does for other components:
https://i.gyazo.com/8427e08ce49131c7c3ecc9a5805a20b9.mp4

Otherwise, the color is dropped, because links use the default text color, if you click them.
https://i.gyazo.com/1fd96a51714ce00281cfbc21f38124ee.mp4

### Proposed changelog entries

* Keep colors when interacting with ionicons as link icon

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade

### Desired reviewers

@janfaracik 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Comment by daniel-beck** [comment]
The second video looks better IMO, as there is an indication that the element was clicked.

Also FYI, GH now supports videos in comments.

**Comment by NotMyFault** [comment]
> The second video looks better IMO, as there is an indication that the element was clicked.

I agree, but I was referring to clicking _on_ elements, because they noticeably blink if you click on them:
https://user-images.githubusercontent.com/13383509/154969820-0541cd6a-559d-4cc6-b3fd-8e68b86295c1.mp4


> Also FYI, GH now supports videos in comments.

Yeah, am aware, this became somewhat of a habit over the years :P

**Review by timja — APPROVED** [review]

**Review by janfaracik — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by daniel-beck** [comment]
> > The second video looks better IMO, as there is an indication that the element was clicked.
> 
> I agree, but I was referring to clicking _on_ elements, because they noticeably blink grey before turning blue again, if you click on them, as opposed to other components: https://user-images.githubusercontent.com/13383509/154969820-0541cd6a-559d-4cc6-b3fd-8e68b86295c1.mp4

All that means is that we need an actual active color that's not grey?

**Review by daniel-beck — APPROVED** [review]
Looks reasonable enough while we don't have an actual active color (explicit approval so comments are not mistaken as blocking feedback).

**Pull request merged**
